projects/go-lib/src/lib/components/go-copy/go-copy.component.ts

```diff
@@ -1,7 +1,35 @@
+import { animate, group, keyframes, state, style, transition, trigger } from '@angular/animations';
 import { Component, ElementRef, HostBinding, Input, ViewChild } from '@angular/core';
 
 @Component({
-  animations: [],
+  animations: [
+    trigger('checkMarkFadeInOut', [
+      state('content_copy', style({
+        opacity: '1'
+      })),
+      state('check', style({
+        opacity: '1'
+      })),
+      transition('content_copy => check', [
+        group([
+          animate('3s', keyframes([
```

Comment:
So I am kind of thinking we can hasten this a bit.. what do you think about cutting this down to `1.5s`?

Reply:
Yeah, coming back to this after awhile, it does feel long. `1.5s` it is 👍